doc/api/v0.yaml

```diff
@@ -224,15 +219,61 @@ components:
           format: date-time
         chain_voteplan_payload:
           type: string
-        fund_id:
-          type: integer
-          format: int32
+
+    ChallengeType:
+      type: string
+      enum:
+        - simple
+        - community-choice
+
+    ProposalWithChallengeInfo:
+      discriminator:
+        propertyName: challenge_type
+        mapping:
+          simple: SimpleProposal
```

Comment:
This should be a fully qualified path to the type (like in `$ref`).

Reply:
Do you think the example [in the spec](http://spec.openapis.org/oas/v3.0.3#discriminator-object) may be misleading?

Reply:
It is. The documentation on swagger.io provides an [example with fully-qualified path](https://swagger.io/docs/specification/data-models/inheritance-and-polymorphism/).

Reply:
I actually think both are valid from the language of the spec, the swagger example just being extra pedantic.